test/integration/helper_test.go

```diff
@@ -22,12 +22,17 @@ import (
 )
 
 const (
-	TestTimeoutMedium        = time.Second * 10
-	TestRetryIntervalMedium  = time.Millisecond * 250
-	TestPlacedGatewayName    = "test-placed-gateway"
-	TestPlacedClusterName    = "test-placed-cluster"
-	TestAttachedRouteName    = "test.example.com"
-	TestAttachedRouteAddress = "172.0.0.3"
+	TestTimeoutMedium         = time.Second * 10
+	ConsistentlyTimeoutMedium = time.Second * 60
+	TestRetryIntervalMedium   = time.Millisecond * 250
+	TestPlacedGatewayName     = "test-placed-gateway"
+	TestPlacedClusterName     = "test-placed-cluster"
+	TestAttachedRouteName     = "test.example.com"
+	TestAttachedRouteAddress  = "172.0.0.3"
+	nsSpoke1Name              = "test-spoke-cluster-1"
+	nsSpoke2Name              = "test-spoke-cluster-2"
+	defaultNS                 = "default"
+	gatewayFinalizer          = "kuadrant.io/gateway"
```

Comment:
the gateway finalizer defined in the `gateway` package. Is it a good idea to have the same value hardcoded in two different places? And would id make sense to keep all the test constants in one [place](https://github.com/Ygnas/multicluster-gateway-controller/blob/integration-test_package/test/util/helper.go#L21-L34)?